test/blocks/aside/mocks/body.html

```diff
@@ -314,12 +315,14 @@ <h2 id="heading-xl-aside-standard-small">Heading XL Aside Standard Small</h2>
     <div class="aside split large">
       <div>
         <div data-valign="middle">
-          <a
-            href="https://images-tv.adobe.com/mpcv3/5286/b4401687-407b-444a-b9f1-9cd169952992_1660079495.1920x1080at3000_h264.mp4#autoplay1">https://images-tv.adobe.com/mpcv3/5286/b4401687-407b-444a-b9f1-9cd169952992_1660079495.1920x1080at3000_h264.mp4#autoplay1</a>
+          <video playsinline="" autoplay="" loop="" muted="">
+            <source src="./drafts/mathuria/aside/media_1491fc79163bea6b7f6daeeb266898344ff827142.mp4" type="video/mp4">
```

Comment:
These sources in the mock will result in 404s in our tests. Is there a way to remove them or are they needed for testing?

Reply:
@JasonHowellSlavin These are holdovers from #979 that I just copy pasted to better align with the new background video functionality, I'm totally fine with removing the sources throughout to clean things up 👍